packages/cli/src/utils.ts

```diff
@@ -200,7 +203,19 @@ export interface Sample {
 
 export async function getTemplates(): Promise<Sample[]> {
   await sampleProvider.fetchSampleConfig();
-  const samples = sampleProvider.SampleCollection.samples.map((sample) => {
+  const version = getVersion();
+  const availableSamples = sampleProvider.SampleCollection.samples.filter(
```

Comment:
do we need consider alpha or beta, not sure gte or lte works fine with version with prerelease postfix.

Reply:
Good point, I tested the `gte()` function with following versions, so I think it should work as expected.
* gte("2.1.0-rc.0", "2.1.0") is false
* gte("2.1.0-rc.0", "2.0.5") is true
* gte("2.1.0-alpha.0", "2.1.0") is false
* gte("2.1.0-alpha.0", "2.0.5") is true